tests/ManageCourses.Tests/UnitTesting/ActionFilter/AcceptTermsFilter.Tests.cs

```diff
@@ -0,0 +1,198 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Security.Claims;
+using System.Threading;
+using GovUk.Education.ManageCourses.Api.ActionFilters;
+using GovUk.Education.ManageCourses.Domain.DatabaseAccess;
+using GovUk.Education.ManageCourses.Domain.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Authentication;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Routing;
+using Moq;
+using NUnit.Framework;
+
+namespace GovUk.Education.ManageCourses.Tests.UnitTesting.ActionFilter
+{
+    [TestFixture]
+    public class AcceptTermsFilterTests    
+    {
+        [Test]
+        public void NoConsent()
+        {
+            var actionExecutingContext = GetActionExecutingContext("foo@bar.com", "Index");
```

Comment:
I tend to avoid actually deliverable addresses in test code
e.g. https://en.wikipedia.org/wiki/.test

minor

Reply:
ok :+1: